packages/panels/src/components/page-header/PageHeader.tsx

```diff
@@ -7,17 +7,20 @@ export interface PageHeaderProps {
   renderBreadCrumbs?: () => ReactNode;
   renderPageHeading?: () => ReactNode;
   renderTabs?: () => ReactNode;
+  TopContent?: ReactNode;
   children?: ReactNode;
 }
 
 export const PageHeader: React.FC<PageHeaderProps> = ({
   renderBreadCrumbs,
   renderPageHeading,
   renderTabs,
+  TopContent,
   children,
 }) => {
   return (
     <Box shadow={"box"} background={cssColor("--lhds-color-ui-50")}>
+      {TopContent}
```

Comment:
Isn't it just possible to put `topContent` above `<PageHeader>...` ?
Does it need to be inside it?